data_models/website.py

```diff
@@ -0,0 +1,35 @@
+"""This script creates a class to store
+information about to a website"""
+
+import validators
+
+from data_models.contents import Contents
+
+
+class Website:
+    """This class stores the information about a website"""
+
+    __LOGO_API = 'https://www.google.com/s2/favicons?domain='
+
+    def __init__(self, url):
+        self.url = url
+        self.is_valid = validators.url(self.url) is True
+        self.domain = self.url.split('/')[2] if self.is_valid else None
+        self.logo_url = self.__LOGO_API + self.domain if self.domain else None
+        self.contents = Contents()
```

Comment:
This can be None. This object(Contents()) is just created and replaced by `set_contents()`.

Reply:
This is anyway get garbage collected.

Reply:
Updated!